src/components/tab-group/tab-group.tsx

```diff
@@ -0,0 +1,71 @@
+import { ComponentInterface, Component, Host, h, Listen, Element, State } from '@stencil/core';
+
+@Component({
+  tag: 'we-tab-group',
+  styleUrl: 'tab-group.scss',
+  shadow: true,
+})
+export class TabGroup implements ComponentInterface {
+  @Element() el: HTMLElement;
+  @State() contentSlot: HTMLDivElement;
+  @State() tabs: NodeListOf<HTMLWeTabElement>;
+  
+  @Listen('tabCallback')
+  tabCallbackHandler(event: any) {
+    const value = event && event.detail;
+    const line: any = this.el.shadowRoot.querySelector('.line');
+
```

Comment:
I think it's better to avoid using any. Please put the type of that event

Reply:
You're right, I replaced any with the right TS type 👍